numba/cuda/cudadrv/runtime.py

```diff
@@ -113,5 +114,23 @@ def get_version(self):
         minor = (rtver.value - (major * 1000)) // 10
         return (major, minor)
 
+    def is_supported_version(self):
+        """
+        Returns True if the CUDA Runtime is a supported version.
+        """
+
+        return self.get_version() in self.supported_versions
+
+    @property
+    def supported_versions(self):
+        """A tuple of supported CUDA toolkit versions. Versions are given in
```

Comment:
```suggestion
        """A tuple of all supported CUDA toolkit versions. Versions are given in
```
added `all` to help imply that numerous versions are going to be returned.